src/options/set.rs

```diff
@@ -192,6 +192,8 @@ pub fn set_options(
     // there (does not emit lines in 1-1 correspondence with raw git output). See #274.
     if opt.color_only {
         opt.side_by_side = false;
+        opt.file_decoration_style = "none".to_string();
+        opt.commit_decoration_style = "none".to_string();
```

Comment:
Hi @ryuta69, thanks for this PR! I have one question: aren't these already set to "none" by the `color-only` feature? (see the way that feature "inherits" from the `raw` feature)

Reply:
@dandavison 
Thank you for review!
Oh, is that supposed to be? If so, currently it seems not working when a user supply some options in gitconfig.

In order to give priority to the options in features/** even when a user supply option in gitconfig, I think we must switch the conditions below.

https://github.com/dandavison/delta/blob/1ca06156570914b85875f95ab6ae1dcf685f082c/src/options/get.rs#L54-L72

Currently, both not working as you suggested.

```
[delta]
	file-decoration-style = ul
# git diff | delta --color-only
# => It still prints file-decoration-style as ul.

------------

[delta]
	minus-style = blue
# git diff | delta --raw
# => It still prints minus-style as blue.
```

Reply:
Hi @ryuta69, sorry to be slow -- I was meaning to look into what we're discussing here but didn't find time yet. Definitely looking forward to having this PR in the next release.

Reply:
@dandavison 
Sure! The first priority is your schedule, and I'll help when you have a time. Again, thank you for this great OSS.

Reply:
OK, this sounds like a potential headache -- let's just keep the hard-coded override to "none" that you have, and we can maybe worry in the future about whether the none-value was supposed to be inherited from the feature automatically.

Reply:
@dandavison 
I understand, thank you for review.
If I find a way to solve, I'm going to create PR.